source/documentation/3.9/rspec-core/RSpec/Core/Example/Procsy.html

```diff
@@ -527,22 +528,11 @@ <h3 class="signature " id="to_proc-instance_method">
 </div>
 
       <div id="footer">
-  Generated on Fri Mar  6 15:02:12 2020 by
+  Generated on Sun Apr  5 16:47:56 2020 by
   <a href="http://yardoc.org" title="Yay! A Ruby Documentation Tool" target="_parent">yard</a>
-  0.9.24 (ruby-2.6.3).
+  0.9.24 (ruby-2.7.0).
 </div>
 
-<script>
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
-  ga('create', 'UA-59095215-1', 'auto');
-  ga('send', 'pageview');
-
-</script>
```

Comment:
I guess we don't need it 👍

Reply:
If @JonRowe is ok to remove Google Analytics?

See comment here https://github.com/rspec/rspec-rails/pull/2298#discussion_r403729246

Sorry I put PR and comments everywhere